### ISSUE TITLE
Unable to find tree.jquery.js.map with collectstatic

### ISSUE THREAD

**Issue by gafderks** [mbraak/django-mptt-admin#505]
If I run `collectstatic` using the latest django_mptt_admin and Django 5.0, I'm getting an error on `django_mptt_admin/tree.jquery.js.map`.

```
92.37 Post-processing 'django_mptt_admin/django_mptt_admin.debug.js' failed!
92.37
92.37 Traceback (most recent call last):
92.37   File "/app/./manage.py", line 21, in <module>
92.37     main()
92.37   File "/app/./manage.py", line 17, in main
92.37     execute_from_command_line(sys.argv)
92.37   File "/app/.venv/lib/python3.12/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
92.37     utility.execute()
92.37   File "/app/.venv/lib/python3.12/site-packages/django/core/management/__init__.py", line 436, in execute
92.37     self.fetch_command(subcommand).run_from_argv(self.argv)
92.37   File "/app/.venv/lib/python3.12/site-packages/django/core/management/base.py", line 412, in run_from_argv
92.37     self.execute(*args, **cmd_options)
92.37   File "/app/.venv/lib/python3.12/site-packages/django/core/management/base.py", line 458, in execute
92.37     output = self.handle(*args, **options)
92.37              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
92.37   File "/app/.venv/lib/python3.12/site-packages/django_gulp/management/commands/collectstatic.py", line 44, in handle
92.37     super(Command, self).handle(*args, **options)
92.37   File "/app/.venv/lib/python3.12/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 209, in handle
92.37     collected = self.collect()
.py", line 249, in converter
92.37     hashed_url = self._url(
92.37                  ^^^^^^^^^^
92.37   File "/app/.venv/lib/python3.12/site-packages/django/contrib/staticfiles/storage.py", line 182, in _url
92.37     hashed_name = hashed_name_func(*args)
92.37                   ^^^^^^^^^^^^^^^^^^^^^^^
92.37   File "/app/.venv/lib/python3.12/site-packages/django/contrib/staticfiles/storage.py", line 425, in _stored_name
92.37     cache_name = self.clean_name(self.hashed_name(name))
92.37                                  ^^^^^^^^^^^^^^^^^^^^^^
92.37   File "/app/.venv/lib/python3.12/site-packages/django/contrib/staticfiles/storage.py", line 143, in hashed_name
92.37     raise ValueError(
92.37 ValueError: The file 'django_mptt_admin/tree.jquery.js.map' could not be found with <django.contrib.staticfiles.storage.ManifestStaticFilesStorage object at 0x7f89fe66ff20>.
```

**Comment by mbraak** [comment]
I'll have a look

**Comment by mbraak** [comment]
This is fixed in version 2.6.1
